SPECS/opam.spec

```
@@ -70,6 +69,9 @@ rm -rf %{buildroot}%{_prefix}/doc
 %{_mandir}/man1/*.1*
 
 %changelog
+* Fri Nov 7 2025 Pau Ruiz Safont <pau.safont@vates.tech> - 2.4.1-1
```

Someday we'll have a linter for that :)

```suggestion
* Fri Nov 07 2025 Pau Ruiz Safont <pau.safont@vates.tech> - 2.4.1-1
```